src/tiles/FeedTile.vue

```diff
@@ -61,8 +62,8 @@
             },
 
             convertDate() {
-                for(var i=0;i<this.feeds.length;i++){
-                    this.feeds[i].publishedAt = new Date(1000*this.feeds[i].publishedAt);
+                for(var i = 0; i < this.feeds.length; i++){
+                    this.feeds[i].publishedAt = new Date(1000 * this.feeds[i].publishedAt);
```

Comment:
IMO you should use moment.js for dates because we use this in other parts of the frontend, but it's up to you!